### DISCUSSION TITLE
Forcing Bindings to Update in WinUI3

### DISCUSSION THREAD

**Discussion by oliverw** [microsoft/microsoft-ui-xaml#8203] [Q&A]
For UWP the solution would be to call `this.Bindings.Update()`. In a WinUI 3 project you even get Intellisense for this, suggesting support is there but then you get a compiler error:

> 'MainWindow' does not contain a definition for 'Bindings' and no accessible extension method 'Bindings' accepting a first argument of type 'MainWindow' could be found (are you missing a using directive or an assembly reference?)

So whats the recommended solution for WinUI 3 to update the views from the binding source?

**Comment by MikeHillberg** [comment]
Have you built the project without that code? The `this.Bindings` field is generated code that's created when the markup compiler runs as part of build. So when you first create a user control it's not there.

**Reply by oliverw** [reply]
Yep, I did multiple times.
